src/firewall-cmd.in

```diff
@@ -2482,6 +2485,9 @@ elif a.complete_reload:
     fw.complete_reload()
 elif a.runtime_to_permanent:
     fw.runtimeToPermanent()
+elif a.reset_to_defaults:
+    fw.config().resetDefaults()
+    fw.reload()
```

Comment:
Decided to reload here instead of at fw_config.py so I do not need to check if running in offline mode, could add a parameter not to attempt to reload though to keep the main logic contained to firewall/core.